### PR TITLE
Afternoon work, dice showing, random number and total score

### PR DESCRIPTION
**Pull request by jadavey91**
Afternoon work, mainly JS.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,41 +1,79 @@
 const playAgainButton = document.querySelector("#playagain");
-const h1 = document.querySelector("#heading");
+const head = document.querySelector("#heading");
 const yourScore = document.querySelector("#score");
 const blankStart = document.querySelector("#blank");
 const diceScore = document.querySelector("#dice");
 const rollButton = document.querySelector("#submit");
```
Comment (nmunro) [inline script.js:6]: Good use of const!

File: script.js [hunk 2]
```diff
@@ -1,41 +1,79 @@
...
-
+// if (randomNumber == 1){
+//     head.textContent= "You Lost"
+//     totScore = 0;
+//     playAgainButton.textContent = "Play Again";
+// }
 
 
 rollButton.addEventListener("click", () => {
+    let randomNumber = Math.ceil(Math.random()* 6 );
```
Comment (nmunro) [inline script.js:18]: Will you occasionally get 0s rolled here? Or does the `ceil` avoid that?

File: script.js [hunk 3]
```diff
@@ -1,41 +1,79 @@
...
+    totScore += randomNumber;
+    yourScore.innerHTML = totScore;
     console.log(randomNumber);
-    totScore = totScore + randomNumber;
+    console.log(totScore);
+    diceScore.src = `img/dice${randomNumber}.png`;
```
Comment (nmunro) [inline script.js:23]: Nice, makes code much easier to read and also gives you much less to have to think about!

File: script.js (hunk 2 above)
Comment (jadavey91) [reply script.js:18]: Ceil rounds up, so 0's are not possible. Math.random()* 6 is creating numbers in the range 0-6 (but not including 0 or 6 ); the ceil is then making those 1 through 6.

File: script.js (hunk 3 above)
Comment (jadavey91) [reply script.js:23]: Genuinely felt like a genius when I removed all the if lines and line 23 worked lol

**Pull request merged**